### DISCUSSION TITLE
How is cluster affected during scaling?

### DISCUSSION THREAD

**Discussion by emoido** [ChistaDATA/Cloud#7] [❓FAQ] [answered]
How is cluster affected during scaling?

**Comment by TheraniAA** [comment, marked as answer]
The ChistaDATA cloud cluster update process depends on a number of its replicas. The single-host cluster requires downtime during maintenance or updates that require a restart. Multi-host clusters can avoid downtime by performing updates sequentially while other hosts handle the workload.
